test/jasmine/tests/mapbox_test.js

```diff
@@ -180,18 +180,26 @@ describe('mapbox credentials', function() {
     });
 
     it('should throw error if token is invalid', function(done) {
+        var cnt = 0;
+
         Plotly.plot(gd, [{
             type: 'scattermapbox',
             lon: [10, 20, 30],
             lat: [10, 20, 30]
         }], {}, {
             mapboxAccessToken: dummyToken
         }).catch(function(err) {
+            cnt++;
             expect(err).toEqual(new Error(constants.mapOnErrorMsg));
```

Comment:
This assertion :arrow_double_up: did not get evaluated before that is this `it` would didn't test anything.

I wish jasmine had the something similar to TAP's `t.plan`.